spec/index.bs

```diff
@@ -25,7 +25,30 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
 Introduction {#intro}
 =====================
 
-This specification rocks.
+*This section is non-normative.*
+
+[Graphics Processing Units](https://en.wikipedia.org/wiki/Graphics_processing_unit), or GPU for short,
+have been essential in enabling rich rendering and computational applications in personal computing.
+WebGPU is an API that exposed the capabilities of GPU hardware for the web.
+The API is designed from the ground up to efficiently map to [Vulkan](https://www.khronos.org/vulkan/),
+[Direct3D 12](https://docs.microsoft.com/en-us/windows/win32/direct3d12/what-is-directx-12-),
+and [Metal](https://developer.apple.com/metal/) native GPU APIs.
+WebGPU is not related to [WebGL](https://www.khronos.org/webgl/) and does not explicitly target OpenGL ES.
+
+WebGPU represents a GPU as a {{GPUDevice}} that that manages resources and a {{GPUQueue}}
```

Comment:
Perhaps worth introducing the idea of adapters here as well?

An adapter represents a piece of physical hardware, while a device is... a context? It's kind of difficult to describe. My wording "logical instantiation of an adapter" isn't very descriptive either.

Reply:
I was hesitating to mention adapters in order to avoid confusion (since this is a non-normative overview). See the follow-up comment for a try.

Reply:
I think it explains it really well, thanks!